js/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte

```diff
@@ -33,39 +34,57 @@
 	export let applyInputSample: (sample: TWidgetExample, opts?: ExampleRunOpts) => void = () => {};
 	export let validateExample: (sample: WidgetExample) => sample is TWidgetExample;
 	export let exampleQueryParams: QueryParam[] = [];
+	export let isDisabled = false;
+	isDisabled = model.inference !== InferenceDisplayability.Yes && model.pipeline_tag !== "reinforcement-learning";
```

Comment:
Should this statement be reactive?

Reply:
it is being changed inside onMount https://github.com/huggingface/hub-docs/blob/c6bd915804c0459aca9e6916e58cadd59b771f37/js/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte#L81-L83

otherwise, there is no need for `$: ` svelte syntax